discovery-provider/plugins/notifications/src/listener.ts

```diff
@@ -52,3 +55,19 @@ export class Listener {
     this.client = null
   }
 }
+
+const getNotification = async (client: Client, notificationId: number): Promise<NotificationRow | null> => {
+  const query = 'SELECT * FROM notification WHERE id = $1 limit 1;'
+  const values = [notificationId] // parameterized query
+  try {
+    const res = await client.query<NotificationRow>(query, values)
+    if (res.rowCount == 0) {
+      logger.warn(`could not find row ${notificationId} in db`)
```

Comment:
don't think this if block is necessary since we handle exceptions anyway

Reply:
oh that's a good point